kratos/geometries/triangle_3d_3.h

```diff
@@ -909,7 +910,53 @@ template<class TPointType> class Triangle3D3
         CoordinatesArrayType point_projected = rPoint;
         const array_1d<double,3> vector_points = rPoint - r_center.Coordinates();
         const double distance = inner_prod(vector_points, normal);
-        if (std::abs(distance) > std::numeric_limits<double>::epsilon()) { // Not in the plane, projecting
+
+        // We check if we are on the plane
+        if (std::abs(distance) > std::numeric_limits<double>::epsilon()) {

```

Comment:
what are u doing here? i don't think this warning belongs to the master...

Reply:
Which warning?

Reply:
Ok, I see